store/src/store.rs

```diff
@@ -341,8 +363,11 @@ impl<T: KeyValueDB> ChainStore for ChainKVStore<T> {
     }
 
     fn get_cell_meta(&self, tx_hash: &H256, index: u32) -> Option<CellMeta> {
-        self.get(COLUMN_CELL_META, &cell_store_key(tx_hash, index))
-            .map(|raw| deserialize(&raw[..]).unwrap())
+        self.get(
+            COLUMN_CELL_META,
+            CellKey::calculate(tx_hash, index).as_ref(),
+        )
+        .map(|raw| deserialize(&raw[..]).unwrap())
```

Comment:
Please use `expect`.
Although I assume the items fetched from the database should be OK. In fact, Sentry has reported several "unwrap" events, which seems related to deserializing database raw data.

Reply:
:cry: I feel a bit wronged. I just keep the old code~ but, that's OK, I will update this after lunch.